src/main/java/javaslang/collection/Tree.java

```diff
@@ -247,7 +247,7 @@ String toString(Tree<T> tree, int depth) {
                         final String children = tree.getChildren()
                                 .map(child -> toString(child, depth + 1))
                                 .join();
-                        return String.format("\n%s%s%s", indent, value, children);
+                        return String.format("%n%s%s%s", indent, value, children);
```

Comment:
```java
 Returns a list string representation of this tree.
     *
     * @return A new string
     */
    default String toLispString() {
```
The javadoc says lis**t**. The name of the method is lis**p**. 
Looking at the code it does look like LISP (produces many braces :-) ).
Is there a typo in the javadoc?